src/fw/drivers/nrf5/rtc.c

```diff
@@ -28,6 +28,10 @@
 #include <stdio.h>
 #include <string.h>
 
+#define OS_RTC_INST NRF_RTC1
```

Comment:
we should likely move this to board?

Reply:
Agreed.  It's a little tricky to put the IRQ handlers together that way for macro reasons, so it's still split between board.c and board.h, but it does work.  6505d85